src/lib/components/modeler/ModelerToMetricsButton.svelte

```diff
@@ -4,7 +4,7 @@
     DerivedModelStore,
```

Comment:
vscode tells me:

`This import is never used as a value and must use 'import type' because 'importsNotUsedAsValues' is set to 'error'.ts(1371)`